### ISSUE TITLE
Dynamic headers and translating

### ISSUE THREAD

**Issue by Hypocrite** [lorenzofox3/Smart-Table#478]
Hi,

I really like your work with Smart-Table. It works really nicely.

But I seem to have a problem when using dynamic table columns and translating them.

The situation I have is that users can select multiple different tests where I need to then display the values in the table.

I have achieved this with the following code:
```
<table st-table="result.events" class="table table-striped">
    <tr>
        <th st-sort="{{col}}" ng-repeat="col in testColumns">{{col}}</th>
    </tr>
    <tr ng-repeat="row in result.events">
        <td ng-repeat="col in testColumns">{{row[col]}}</td>
    </tr>
</table>
```

This works fine when using the original English values in it.

But I would need to translate the values to Finnish and I am using a gettext plugin for that:
https://github.com/rubenv/angular-gettext

So basically I would need to change:
```
 <th st-sort="{{col}}" ng-repeat="col in testColumns">{{col}}</th>
```

to

```
 <th st-sort="{{col}}" ng-repeat="col in testColumns">{{translate(col)}}</th>
```

This translates correctly but then it does not show the results anymore.

I could also translate the values in controller but the values are usually something like Nopeus 5m, which then gives me this error:
Error: [$parse:syntax] Syntax Error: Token '5' is an unexpected token at column 8 of the expression [Nopeus 5m] starting at [5m].

Thanks for your help!

**Comment by Hypocrite** [comment]
I was able to fix this with st-safe-src.

**Comment by lorenzofox3** [comment]
:+1:
